api/aks/struct.go

```diff
@@ -6,11 +6,11 @@ import (
 )
 
 type AzureProvider struct {
-	User string
-	Cluster     *Cluster
-	HACluster   *HACluster
-	Spec        utils.Machine // do we really need this over here or keeping it inside cluster/HAcluster level makes more sense
-	Credentials *Cred
+	User           string
+	Cluster        *Cluster
+	HACluster      *HACluster
+	Spec           utils.Machine // do we really need this over here or keeping it inside cluster/HAcluster level makes more sense
+	Credentials    *Cred
 	ResourceGroups armresources.ResourceGroupsClientCreateOrUpdateResponse //can remove it, temp
 }
```

Comment:
@AvineshTripathi you can move this structure to utils where it first originated as the structure will be same